### DISCUSSION TITLE
enquiry

### DISCUSSION THREAD

**Discussion by RimshaNaseem** [community/community#146459] [GitHub Education]
### Hi everyone,

didnt get student pack yet

**Comment by RimshaNaseem** [comment]
does someone know how to activatecopilot account

**Comment by davevad93** [comment]
Hi @RimshaNaseem ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @RimshaNaseem was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by RimshaNaseem** [reply]
will you please tell me how to link copilot with r br=ecause its not working there they said youare signed in but not activated the account how should i activate my github account im student

**Reply by davevad93** [reply]
Hey @RimshaNaseem as I said above, if your status isn't purple you can't link your profile with Copilot, in which case you have to wait, then you can redeem your offer by going to the [pack offers page](https://education.github.com/pack), scroll down to the "*All offers*" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student)
